index.html

```diff
@@ -503,6 +507,204 @@
             </div>
           </div>
         </li>
+        <li> <!--HW 01-->
+          <div class="row" data-tag="">
+            <div class="column">Define a Universal Set</div>
+            <div class="column">$\Omega = \{1, 2, 3, 4, 5, 6\}$</div>
+            <div class="column">
+              <code> \Omega = \{1, 2, 3, 4, 5, 6\} </code>
+            </div>
+          </div>
+        </li>
+        <li> <!--HW 02-->
+          <div class="row" data-tag="">
+            <div class="column">Use Regular Text in Math</div>
+            <div class="column">$\text{Outcomes of a Six-Sided Die} = \{1, 2, 3, 4, 5, 6\}$</div>
+            <div class="column">
+              <code> \text{Outcomes of a Six-Sided Die} = \{1, 2, 3, 4, 5, 6\} </code>
+            </div>
+          </div>
+        </li>
+        <li> <!--HW 03-->
+          <div class="row" data-tag="">
+            <div class="column">Math Blackboard Font</div>
+            <div class="column">$\mathbb{A B C D E F G H I J K L M N O P Q R S T U V W X Y Z}$</div>
+            <div class="column">
+              <code>\mathbb{A B C D E F G H I J K L M N O P Q R S T U V W X Y Z}</code>
+            </div>
+          </div>
+        </li>
+        <li> <!--HW 04-->
+          <div class="row" data-tag="">
+            <div class="column">Subset Equal</div>
+            <div class="column">$\subseteq$</div>
+            <div class="column">
+              <code>\subseteq</code>
+            </div>
+          </div>
+        </li>
+        <li> <!--HW 05-->
+          <div class="row" data-tag="">
+            <div class="column">Math Caligraphic Font</div>
+            <div class="column">$\mathcal{A B C D E F G H I J K L M N O P Q R S T U V W X Y Z}$</div>
+            <div class="column">
+              <code>\mathcal{A B C D E F G H I J K L M N O P Q R S T U V W X Y Z}</code>
+            </div>
+          </div>
+        </li>
+        <li> <!--HW 05-->
+          <div class="row" data-tag="">
+            <div class="column">Dots \ Elipses</div>
+            <div class="column">$\dots$</div>
+            <div class="column">
+              <code>\dots</code>
+            </div>
+          </div>
+        </li>
+        <li> <!--HW 05-->
+          <div class="row" data-tag="">
+            <div class="column">Center Dots \ Elipses</div>
+            <div class="column">$\cdots$</div>
```

Comment:
What do you think of adding an example for when `\cdots` is used, over say `\ldots`?